api/handlers.go

```diff
@@ -30,6 +31,9 @@ import (
 
 const InvalidParamsMessage = "Invalid query or path parameters"
 
+// jetIDRegexp uses for a validation of the JetID
+var jetIDRegexp = regexp.MustCompile(`^[0-1]{2,64}$`)
```

Comment:
why starts from 2 and not one?

Reply:
[0-1] - means can be only 1 or 2 
{2,64} - means minimum 2 symbols, maximum 64. because jetID can't be a single

Reply:
But why jetID can't be a single? "0" or "1" is possible jetID too

Reply:
yep yep. you right.